src/core/etl/src/Flow/ETL/Row/Entry/Type/NativeObject.php

```diff
@@ -2,11 +2,11 @@
 
 declare(strict_types=1);
 
-namespace Flow\ETL\Row\Entry\TypedCollection;
+namespace Flow\ETL\Row\Entry\Type;
 
 use Flow\ETL\Exception\InvalidArgumentException;
 
-final class ObjectType implements Type
+final class NativeObject implements Type
```

Comment:
yeah, the reason why I kept ObjectType was that Object is a reserved keyword which IMO sounds better than NativeObject - that's why I also left ScalarType. 

`ObjectType::of(\DateTimeImmutableClass)` also looks more descriptive than `ObjectType::fromString()` - maybe `ObjectType::fromClass()` would be better.

Reply:
TBH. I have hard to swallow problem with the `ObjectType` itself, I cannot find a real case for its usage in my head, as from my perspective rather converter(?) alike, than real type, especially when looking at the parquet [definition of logical types](https://github.com/apache/parquet-format/blob/master/LogicalTypes.md#lists) where value is just converted to binary, where here it would (probably) string.

Reply:
Map of ints to \DateTimeImmutable is an example of where ObjectType would be an type of Map Element, same with Lists, for example List of Uuid's

Reply:
Restored old name.